projects/packages/connection/legacy/class-jetpack-ixr-client.php

```diff
@@ -122,4 +132,17 @@ public function get_jetpack_error( $fault_code = null, $fault_string = null ) {
 
 		return new \WP_Error( "IXR_{$fault_code}", $fault_string );
 	}
+
+	/**
+	 * Retrieve a response header if set.
+	 *
+	 * @param  string $name  header name.
+	 * @return string|bool Header value if set, false if not set.
+	 */
+	public function get_response_header( $name ) {
```

Comment:
Sometimes headers get lowercased. I'd suggest checking both in here.

Reply:
Thanks Aidvu, I was unaware of this and will update appropriately

Reply:
They are case-insensitive: https://stackoverflow.com/a/41169947... but in reality, it depends. Best is to lowercase the keys, and search for a lowercase header.